packages/eservice-template-process/src/routers/EServiceTemplateRouter.ts

```diff
@@ -814,13 +814,14 @@ const eserviceTemplatesRouter = (
         try {
           validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE, M2M_ADMIN_ROLE]);
 
-          const updatedEServiceTemplate =
+          const { metadata, data: updatedEServiceTemplate } =
```

Comment:
Unrelated change, same for other changes to tempalte process, they are due to base branch change probably, let's revert all of them!